sentry_sdk/scope.py

```diff
@@ -441,13 +441,21 @@ def clear(self):
 
     @_attr_setter
     def level(self, value):
-        # type: (Optional[LogLevelStr]) -> None
-        """When set this overrides the level. Deprecated in favor of set_level."""
+        # type: (LogLevelStr) -> None
+        """
+        When set this overrides the level. Deprecated in favor of set_level.
+
+        :param value: The level to set.
+        """
         self._level = value
```

Comment:
Should we have a deprecation warning?

Reply:
Good idea. Added it. (turns out it has been deprecated since 1.0 :-) )

Reply:
Great! Have we deleted it in 2.0? If not, we could